v0/util/index.js

```diff
@@ -1546,7 +1546,10 @@ const isOAuthSupported = (destination, destHandler) => {
 
 function isAppleFamily(platform) {
   const appleOsNames = ["ios", "watchos", "ipados", "tvos"];
-  return appleOsNames.includes(platform?.toLowerCase());
+  if (typeof platform === "string") {
```

Comment:
Is there a need to support another data type (like array) with valid platform value coming in ?